src/form/MapSelectAssetOrPoint.tsx

```diff
@@ -78,61 +81,55 @@ const MapSelectAssetOrPoint = (props:Props) => {
     };
   }, [rasterUuid]);
 
-  const setLocation = (location: Location | null) => {
-    if (location !== null && raster !== null) {
-      // Check if location fits within the raster
-      const { lat, lng } = location;
-      const bounds = raster.spatial_bounds;
-
-      if (!bounds) return;
-
-      const inBounds = (
-        lat >= bounds.south && lat <= bounds.north &&
-        lng >= bounds.west && lng <= bounds.east
-      );
-
-      if (!inBounds) return;
-    };
-    valueChanged({ asset: null, location: location});
-  };
-
-  const chooseLocation = !!setLocation;
-
-  const setAsset = (option: { value: Asset } | null) => {
-    const asset = option && option.value;
-    if (asset && asset.view) {
-      const lat = asset.view[0];
-      const lng = asset.view[1];
-      if (raster !== null) {
-        if ( !raster.spatial_bounds) {
-          return;
-        }
-        const inBounds = (
-          lat >= raster.spatial_bounds.south && lat <= raster.spatial_bounds.north &&
-          lng >= raster.spatial_bounds.west && lng <= raster.spatial_bounds.east
-        );
-        if (!inBounds) { 
-          return; 
-        }
+  // const setLocation = (location: Location | null) => {
+  //   if (location !== null && raster !== null) {
+  //     // Check if location fits within the raster
+  //     const { lat, lng } = location;
+  //     const bounds = raster.spatial_bounds;
+
+  //     if (!bounds) return;
+
+  //     const inBounds = (
+  //       lat >= bounds.south && lat <= bounds.north &&
+  //       lng >= bounds.west && lng <= bounds.east
+  //     );
+
+  //     if (!inBounds) return;
+  //   };
+  //   valueChanged({ asset: null, location: location});
+  // };
+
+  // const chooseLocation = !!setLocation;
+
+  // const setAsset = (asset: Asset | null) => {
+  //   if (asset && asset.location) {
+  //     const { lat, lng } = asset.location;
+  //     // if (raster !== null) {
+  //     //   if ( !raster.spatial_bounds) {
+  //     //     return;
+  //     //   }
+  //     //   const inBounds = (
+  //     //     lat >= raster.spatial_bounds.south && lat <= raster.spatial_bounds.north &&
+  //     //     lng >= raster.spatial_bounds.west && lng <= raster.spatial_bounds.east
+  //     //   );
+  //     //   if (!inBounds) { 
+  //     //     return; 
+  //     //   }
+  //     // }
+  //     valueChanged({ asset: asset, location: {lat ,lng}});
+  //   } else {
+  //     valueChanged({ asset: null, location: null});
+  //   }
+  // };
```

Comment:
Commented these codes out for now. After this one is merged, I will continue with the ticket to make this component reusable by the RasterAlarmForm and the GeometryField as well.